### ISSUE TITLE
Decide on an interface

### ISSUE THREAD

**Issue by snarisi** [Yipit/puppy#3]
I haven't been very consistent so far but most have followed the puppeteer API, but with snake casing and using kwargs insteads of options dicts. We should clean this up and decide whether to

- mimic the puppeteer interface exactly,
- make a "pythonic" version of the puppeteer interface, or
- do something else entirely

cc @spulec @annieholladay @psarma89

**Comment by annieholladay** [comment]
I think i'd agree with steve to make it pythonic. since this is supposed to be a python interface to puppeteer it probably shouldnt matter to puppy's api what puppeteer was written with or it's conventions

**Comment by psarma89** [comment]
I would agree with the pythonic version, I think trying to mimic puppeteer interface (pypeteer) was confusing in development and in usage for our current ypuppet implementation. Edge cases came up especially with the snake casing and kwargs that I would prefer not to repeat.
